src/ir/function.rs

```diff
@@ -131,7 +131,8 @@ pub fn cursor_mangling(ctx: &BindgenContext,
     }
 
     // Try to undo backend linkage munging (prepended _, generally)
-    if cfg!(target_os = "macos") {
+    if get_abi(cursor.cur_type().call_conv()).map_or(true, |abi| abi != abi::Abi::Stdcall) &&
```

Comment:
I don't think its a fix either. In fact, I don't fully understand the purport of this if statement since I    got the proper names on macos when the calling convention is `stdcall` or `cdecl` or whatever... I'll remove this line.